### DISCUSSION TITLE
6 Slot Adapter V2

### DISCUSSION THREAD

**Discussion by Mythris29** [sanni/cartreader#537] [Need help]
I ordered some of the 6-slot adapter boards v2 and after making 2 of them both don't let me dump CiC required (SA1) games. I tried to look through the differences on the schematics but it's over my head...I guess it's possible that I messed up making 2 of them but my v1 board works fine. I'm also able to dump non SA1 super nintendo games just fine.

So far I haven't seen anybody who can confirm they've gotten a v2 board and it works so I just wanted to open this up and make sure there isn't an issue with the design of the board itself.

**Comment by Mythris29** [comment]
I'm on HWv5

**Comment by ccfman2004** [comment]
I ordered some rev2 boards and are waiting for them to arrive. Once they do and I can test one I will update.

**Comment by Mythris29** [comment]
> I ordered some rev2 boards and are waiting for them to arrive. Once they do and I can test one I will update.

Awesome, I'm not ruling out that I'm the problem here but I'd like to at least see some others attempt to build it and see if it works or not :)

**Comment by sanni** [comment]
To check a PCB for errors with Kicad installed open the *.pro file of the PCB design and then in the newly opened KiCad window double click on both *.kicad_pcb and *.kicad_sch(1) and arrange the two new windows side by side. Then in the schematic window use the highlight tool(2) and click on a pin(3). The traces connected to this pin will now be highlighted in the PCB window(4) and you can use a multimeter with your real-world PCB to check for proper connection.

![9494949493215154](https://user-images.githubusercontent.com/19169494/192794881-fa7cd0e8-a182-461e-95f0-2cc2d334aeac.jpg)

**Comment by ccfman2004** [comment]
My order of rev 2 six slot adapters have arrived and I will start building it later today and get back.

**Reply by Mythris29** [reply]
Thank you!

**Comment by ccfman2004** [comment]
I assembled a REV 2 board and had no issues dumping my Power Rangers Racing game which is an SA1 game. Power cycled several times and re-inserted game several times and it dumped the game correctly every time. I even programmed a new PIC12F629 chip and assembled a new CIC board and still no issues. 

Are you certain the board you are having issues with is in fact a REV 2 board. The silkscreen on the back should say the revision number. If it is a REV 2 board and you are sure the CIC PCB works and have double checked the SNES slot solder joints you may have a defective board. 

I was having issues dumping Famicom games after several successful dumps and it turned out one of the pins had a flakey solder joint that I missed several times checking continuity. After fixing that I had issues dumping Famicom games.

**Reply by Mythris29** [reply]
Thank you! Sounds like I'll need to look into my boards and where I messed up the soldering.
